packages/multicast_dns/lib/src/lookup_resolver.dart

```diff
@@ -25,7 +25,7 @@ class PendingRequest extends LinkedListEntry<PendingRequest> {
   final StreamController<ResourceRecord> controller;
 
   /// The timer for the request.
-  Timer timer;
+  late Timer timer;
```

Comment:
maybe make `_timer` `nullable` instead.  This way, a consumer of this class can check if the value is already set, etc..